docs/prow/dependency-update.md

```diff
@@ -0,0 +1,68 @@
+# Dependency updates
+
+To help keep dependency up to date you can use dependabot together with automated testing and merging of pull requests.
+
+Dependabot is a GitHub tool which can monitor repository and create pull requests to update dependencies. Dependabot support multiple languages. Check dependabot documentation for details.
+
+Pull requests created by dependabot will be tested by Prow. Pull requestes with required tests passed and meeting criteria can be automatically merged by Tide without human review.
+
+Owners of updated files can subscribe to receive notifications when pull request is merged without human review.
+
+## How to use it
+
+Enable dependabot in your repository. [Here](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates) is detailed dependabot documentation.
```

Comment:
```suggestion
1. Enable Dependabot in your repository. Learn more under [About Dependabot version updates](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates).
```
What exactly do we want users to do in this step? Read the Dependabot instructions and enable Dependabot on their own? If yes, what's the purpose of the following step?